README.md

```diff
@@ -624,6 +624,64 @@ all of the previous, including applying the credited balance to the Order.
 This does not apply to the `$subscription->swapNextCycle('other-plan')`, which simply waits for the next billing cycle
 to update the subscription plan. A common use case for this is downgrading the plan at the end of the billing cycle. 
 
+### How can I load coupons and/or plans from database?
+
+Because Cashier Mollie uses contracts a lot it's quite easy to extend Cashier Mollie and use your own implementations.
+You can quite easily load coupons/plans from database, a file or even a JSON API.
+
+For example a simple implementation of plans from the database:
+
+Firstly you create your own implementation of the plan repository and implement `Laravel\Cashier\Plan\Contracts\PlanRepository`.
+Implement the methods according to your needs and make sure you'll return a `Laravel\Cashier\Plan\Contracts\Plan`.
+
+```php
+use App\Plan;
```

Comment:
Implying an Eloquent model here. I think we should be explicit here, perhaps include it as an example.

This would also clear up the `return $plan->buildCashierPlan();` part.

Reply:
I've added an example Plan model. I thought it was somewhat less important, that's why I put it in a collapsible section.

Hope it's clear what's happening to give other people an idea where to start when implementing your own PlanRepository. It's quite tricky to not go too much into detail, otherwise the documentation will only be focused on using Eloquent models. The section is already quite large. If more documentation is needed I'll probably move it to a file like `docs/contracts.md`.